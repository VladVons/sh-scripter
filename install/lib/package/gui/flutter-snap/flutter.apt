openjdk-8-jdk

qemu-kvm 
libvirt-daemon-system 
libvirt-clients 
bridge-utils

adb
