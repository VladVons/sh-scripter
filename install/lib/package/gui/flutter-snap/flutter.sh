PostInstall()
{
    usermod -aG kvm $cUser
}
