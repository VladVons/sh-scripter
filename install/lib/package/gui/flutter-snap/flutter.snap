flutter --classic
android-studio --classic
code --classic
